Purchases/Public/RCPurchases.h

```diff
@@ -392,12 +399,21 @@ NS_SWIFT_NAME(entitlements(_:)) RC_UNAVAILABLE("entitlements: has been replaced
 
 #undef RC_UNAVAILABLE
 
-/**
- Invalidates the cache for purchaser information.
- This is useful for cases where purchaser information might have been updated outside of the app, like if a
- promotional subscription is granted through the RevenueCat dashboard.
- */
-- (void)invalidatePurchaserInfoCache;
+@end
+
+@interface RCPurchases (SubscriberAttributes)
+
+#pragma mark Subscriber Attributes
+
+- (void)setAttributes:(NSDictionary<NSString *, NSString *> *)attributes;
```

Comment:
is it possible to do something like `NSDictionary<NSString *, nullable NSString *> *`?

Reply:
`NSDictionary` is alergic to nil. It can, however, have `NSNull`. That may complicate things, though. 
I'll see if I can find something nice and clean to do here, though.